### PR TITLE
Fixed typo preventing correct nev to be querried for file format version

### PR DESCRIPTION
**Pull request by mdenker**
This PR fixes a small typo/bug in the BlackrockIO of Neo.

### CODE REVIEW

File: neo/io/blackrockio.py [hunk 1]
```diff
@@ -391,7 +391,7 @@ def __extract_nev_file_spec(self):
         """
         Extract file specification from an .nsx file
```
Comment (JuliaSprenger) [inline neo/io/blackrockio.py]: There is also a typo in the docstring here

Comment (mdenker) [reply neo/io/blackrockio.py]: True... fixing

**Comment by samuelgarcia** [comment]
@mdenker: make teh same patch but for the file blackrockio_v4.py wich is now the old BlackrockIO.

Otherwise we have a big conflict here since rawio merge.

**Comment by mdenker** [comment]
Hi @samuelgarcia 
I transferred the changes to v4 blackrockio, and to the rawio of Blackrock, which had the same bug.

**Comment by samuelgarcia** [comment]
OK. thanks can I merge ?

**Comment by mdenker** [comment]
Yes please go ahead. I'm very confident this small fix is ok.

**Pull request merged**